_posts/2019-12-24-templatowanie-jobow-jenkinsa.md

```diff
@@ -0,0 +1,115 @@
+---
+layout:    post
+title:     "Automat dodający joby do Jenkinsa"
+published: false
+date:      2019-12-24 08:00:00 +0100
+author:    dkubiak
+tags:
+    - jenkins job
+    - jenkins template
+    - devops
+    - jenkins-job-builder
+    - jenkins-jobs
+---
+## Zastosowanie
+
+W ogarniającym nas świecie mikroserwisów skala projektów do utrzymania staje się ogromna. Każdy z tych projektów musimy przecież: zbudować, przetesować, zdeployować itd. Przy liczbie projektów 20+ przestaje to być trywialne. W tym artykule zajmniemy się pierwszym zagadnieniem, automatyzacją buildow, jednak opisany tutaj sposób bez problemu można zastosować do innych aspektów.
+
+Do budowania projektów 20+ zazwyczaj staramy się utrzymać jedno narzędzie i to oczywiście działa na plus. W tym artykule posługiwać będziemy się mavenem.
```

Comment:
Do budowania naszych projektów 20+  - to trochę dziwnie brzmi jak 20 letnie i starsze. Może "Przy większej liczbie..."

Reply:
zmieniam to na: "Do budowania projektów starajmy się używać jednego narzędzia, ogromnie wpłynie to na proces unifikacji. W tym artykule posługiwać będziemy się mavenem."

Reply:
wywalilem wszytskie 20+ intencja była 500+ ale nie mieszajmy tutaj polityki ;)

Reply:
to daj chociaż 300+ :P

Reply:
@dkubiak @glipecki co tutaj ostatecznie?